app/controllers/admin/add_user_to_group_controller.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Admin
+  class AddUserToGroupController < ApplicationController
+    include Admin::RestrictAccessToAdminMode
+
+    before_action :login_required
+
+    def new; end
+
+    def create
```

Comment:
Okay, I think I'm starting to get it.  If I ignore "new" and "create" and read them instead as "GET" and "POST" and then interpret "controller" as "action", then this all maps pretty well from the old system.  It gets a _little_ confusing when there are non-crud actions mixed in (like /account/verifications/resend).  I'm not recommending this, but would another approach for things like that be to do yet another layer of namespacing?  e.g., /account/verifications/resend_controller?  (Just for the purpose of understanding and getting used to this way of doing routing...)

Reply:
yes, exactly — in fact i did stuff like that on the next PR for Observations/Namings/Votes. 

With the routes for that kind of nesting, you can configure the nested `resource` with `shallow: true` so that the route only gives as many levels as necessary to specify the right record — also, so you're not trying to grab and pass around an ID at each level of nesting. That was a big conceptual hurdle on my attempt at namespacing NameDescriptions, for example, i thought i had to make all these methods pass a `parent_id`. Now that i've understood this, those should be easier to deal with.

`verifications/resend` was such a simple action it didn't seem worth it but that's the idea.

Reply:
Understood, thanks!